Automattic/wp-calypso
# Performance Profiler: Center screenshot only in mobile view

## Proposed Changes

* Update the media query to center the screenshot only in `mobile` view.
* For viewports greater than mobile, the screenshot will be left-aligned when it wraps to next line

Further context: p1726494298986469/1726494143.357009-slack-C02JPCHEKDY

#### Mobile tab

https://github.com/user-attachments/assets/f9f0dbd5-8caf-40c5-816b-407bf7660d5c




#### Desktop tab

https://github.com/user-attachments/assets/4def9bb0-2b35-43b4-852c-fa0dec28dad5






## Why are these changes being made?


* To fix some issues with the current view

## Testing Instructions



* Open the Performance Tool and resize the screen
* Check that when the screenshot wraps to the next line, it aligns correctly to the left.
* Check when reaching the mobile view, it centers
* Check it approximately matches suggestions in [this comment](https://github.com/Automattic/wp-calypso/pull/94551#issuecomment-2355011832)

## Pre-merge Checklist
